content/micropython/00.first-steps/03.runtime-package/runtime-package.md

```diff
@@ -0,0 +1,367 @@
+---
+title: "Arduino MicroPython Runtime"
+description: "Learn how to use the Arduino MicroPython runtime package, which allows you to write MicroPython code in a familiar Arduino style."
+author: "Karl Söderby"
+tags: [MicroPython, Runtime]
+micropython_type: test
+---
+
+The [Arduino Runtime Package](https://github.com/arduino/arduino-runtime-mpy/tree/main) is a MicroPython package that allows you to write and program your board using the classic `setup()` and `loop()` constructs.
+
+The package was designed to make it easier to create programs, particularly for those familiar with the Arduino C++ environment.
+
+In this tutorial, you will learn how the package works, along with a set of examples that will get you started.
+
+## Requirements
+
+To follow this tutorial, you will need to have the following requirements ticked:
+
+### Hardware Requirements
+
+- [A MicroPython compatible board](/micropython/first-steps/install-guide/#micropython-compatible-arduino-boards) (in this tutorial, we will be using an [Arduino Nano ESP32](https://store.arduino.cc/products/nano-esp32))
+- MicroPython installed on your board (see [installation instructions for MicroPython](/micropython/first-steps/install-guide/)).
+
+### Software Requirements
+
+- [Arduino Lab for Micropython](https://labs.arduino.cc/en/labs/micropython) - an editor where we can create and run MicroPython scripts on an Arduino board.
+- [Arduino MicroPython Package Installer](https://labs.arduino.cc/en/labs/micropython-package-installer) - for installing **MicroPython packages** on an Arduino board.
+
+## Installation
+
+To use the runtime package, we will need to install it first.
+
+1. Download and install the [Arduino MicroPython Package Installer](https://labs.arduino.cc/en/labs/micropython-package-installer).
+2. Connect your board to your computer.
+3. Run the tool. In the tool, you should now see your board connected.
+
+   ![Board connected.](assets/connect-device.png)
+
+4. After verifying that your board is connected, click on the search field, and search for **runtime**. Install the package.
+
+   ![Install the package.](assets/install-package.png)
+
+5. When the installation is complete, we are ready to use the package.
```

Comment:
Try to always use "library", a package is usually comprised by a series of modules and it's more difficult to grasp in this context.